packages/api/src/middleware/adminMiddleware.ts

```diff
@@ -0,0 +1,29 @@
+import { NextFunction, Request, Response } from 'express';
+import { Admin } from '@hangar/database';
+
+export const adminMiddleware = async (req: Request, res: Response, next: NextFunction) => {
+  let userAdmin;
+  try {
+    userAdmin = await req.entityManager.findOne(Admin, { user: req.user?.id });
+  } catch {
+    res.sendStatus(500);
+    return;
+  }
+
+  if (!userAdmin) {
+    // Admin does not exist in the database
+    res.sendStatus(403);
+    return;
+  }
+
+  next();
+};
+
+/**
```

Comment:
Can you move this comment up on top of the method itself; it won't be detected as a JSDoc comment down here. You'll know if it worked if you hover over `adminMiddleware` and you see your comment